src/app/(sidebar)/endpoints/[[...pages]]/page.tsx

```diff
@@ -116,6 +127,106 @@ export default function Endpoints() {
   const [urlPathParams, setUrlPathparams] = useState("");
   const [urlParams, setUrlParams] = useState("");
 
+  const getRpcPostPayloadProps = (endpoint: string) => {
+    const defaultRpcRequestBody = {
+      jsonrpc: "2.0",
+      id: 8675309,
+      method: pageData?.rpcMethod,
+    };
+
+    switch (endpoint) {
```

Comment:
I wanted to move this to `helpers/` because it's gotten so long but it uses component level props so I left it here

Reply:
I don't mind it here. :+1: